src/ui/settings/index.js

```diff
@@ -29,7 +29,8 @@ export default function setupSettings(kbnServer, server, config) {
     return client
       .get({ ...clientSettings })
       .then(res => res._source)
-      .then(user => hydrateUserSettings(user));
+      .then(user => hydrateUserSettings(user))
+      .catch(e => { return {}; });
```

Comment:
You can do `.catch(e => ({}));` instead. While `.catch(e => {})` would result in an empty block function, `({})` is treated as an expression resolving to an empty object.